packages/ffe-datepicker-react/src/calendar/Header.js

```diff
@@ -1,50 +1,139 @@
 import React from 'react';
 import { func, number, string } from 'prop-types';
-import ChevronIkon from '@sb1/ffe-icons-react/lib/chevron-ikon';
+import { SecondaryButton } from '@sb1/ffe-buttons-react';
+import Dropdown from '@sb1/ffe-dropdown-react';
+import MonthPicker from './MonthPicker';
+import i18n from '../i18n/i18n';
 
 export default function Header(props) {
     const {
         datepickerId,
         month,
+        year,
+        language,
         nextMonthHandler,
-        nextMonthLabel,
         previousMonthHandler,
-        previousMonthLabel,
-        year,
+        todayHandler,
+        selectYearHandler,
     } = props;
 
+    function renderYearOptions() {
+        const distribution = 5;
+        const minYear = year - distribution;
+        const maxYear = year + distribution;
+        const options = [];
+        let currentYear = minYear;
+        let option;
+
+        while (currentYear <= maxYear) {
+            option = (
+                <option key={currentYear} value={currentYear}>
+                    {currentYear}
+                </option>
+            );
+            options.push(option);
+            currentYear += 1;
+        }
+
+        return options;
+    }
+
+    const todayLabel = i18n[language].TODAY;
+
     return (
-        <div className="ffe-calendar__header">
-            <div className="ffe-calendar__header-inner-wrapper">
-                <button
-                    className="ffe-calendar__month-nav ffe-calendar__previous"
-                    onClick={previousMonthHandler}
-                    tabIndex="-1"
-                    title={previousMonthLabel}
-                    type="button"
-                >
-                    <ChevronIkon className="ffe-calendar__icon-prev" />
-                </button>
-                <header
-                    aria-live="assertive"
-                    aria-atomic="true"
-                    className="ffe-calendar__title"
-                    id={`${datepickerId}-title`}
-                >
-                    <div id={`${datepickerId}__month-label`}>
-                        <span className="ffe-calendar__month">{month}</span>
-                        <span className="ffe-calendar__year">{year}</span>
+        <div id={`${datepickerId}-header`} className="ffe-calendar__header">
+            <div
+                id={`${datepickerId}-settings`}
+                className="ffe-calendar__header-container"
+            >
+                <div className="ffe-calendar__header-wrapper">
+                    <div className="ffe-calendar__header-define" />
+                    <div className="ffe-calendar__header-break ffe-calendar__header-break-1" />
+                    <div className="ffe-calendar__header-block" />
+                    <div className="ffe-calendar__header-define" />
+                    <div className="ffe-calendar__header-break ffe-calendar__header-break-2" />
+                    <div className="ffe-calendar__header-block" />
```

Comment:
I don't really understand what's going on here.
Is all of this working well across browsers, and also from a UU perspective?

Edit: I read in your docs this should have good browser support.

Reply:
As I wrote in the initial PR message;

I think a better solution for breakpoints going forward would be css-element-queries. Read more about it here; https://github.com/marcj/css-element-queries.

I just did not want to implement css-element-queries without knowing more about the project, so I decided on an independent solution with native css and html instead. Mostly to demonstrate how the solution would look. I don’t recommend using this technique globally since there will be an excessive amount of html markup.